### PR TITLE
Refactor search inputs

### PR DESCRIPTION
**Pull request by jpople**
Closes unticketed

### Description Of Changes

Styling tweaks to D&D search input "clear" button (particularly hover state).  Before:
Screenshot 2024-10-18 at 09 38 09
After:
Screenshot 2024-10-18 at 12 31 35

### Code Changes

* Refactor D&D's debounced search input to use existing SearchBar component
* Adds new "neutral" colors to Tailwind config

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [ ] Issue Requirements are Met
* [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: clients/admin-ui/tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,23 @@
 module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx,mdx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        // TODO: unify with existing Tailwind "gray" color
```
Comment (gilluminate) [inline clients/admin-ui/tailwind.config.js:7]: Is there a ticket for this, by chance?

**Review by gilluminate — APPROVED** [review]
Looks good. What's the plan for making these consistent across the board? Are we thinking it'll be part of the Inputs migration?

File: clients/admin-ui/tailwind.config.js (hunk 1 above)
Comment (jpople) [reply clients/admin-ui/tailwind.config.js:7]: I'm thinking this should be able to be done easily as part of the larger colors migration, but I can ticket it separately if you think that's a good idea.

Comment (gilluminate) [reply clients/admin-ui/tailwind.config.js:7]: Is that what @Kelsey-Ethyca is working on? Just want to make sure it doesn't get lost in the shuffle.

**Pull request merged**